### ISSUE TITLE
Running Out of Memory

### ISSUE THREAD

**Issue by Pratyush1991** [OpenDroneMap/WebODM#172]
While trying to process a dataset of 174 images taken by iphone5s but it runs out of memory. At first I tried processing by assigning 12GB of RAM and with 6 cores and then with 14GB RAM and 7 cores. In both the cases the process ran out of memory which I feel should be enough to process 174 images. So I'm not sure what is the problem. The error log as well as link to my data set have been attached below.
I will be glad if you could help..:-)
  
https://drive.google.com/drive/folders/0B7tV_P5gZivFYkJPTUtieTBJNzg?usp=sharing

![mem](https://cloud.githubusercontent.com/assets/28738918/26202631/4d5e69f0-3c02-11e7-993c-32215c8e325a.png)

	Blending texture patches 100%... done. (Took 20.075s)
Generating texture atlases:
	Sorting texture patches... done.
done.

	Working on atlas 41 100%... done.
	Finalizing texture atlases... done. (Took: 3.698s)
Building objmodel:
	Saving model... 	Saving model... 	Saving model... done.
Whole texturing procedure took: 88.492s
[INFO]    Running ODM Texturing Cell - Finished
[INFO]    Running ODM Georeferencing Cell
[DEBUG]   None
[WARNING] No coordinates file. Generating coordinates file: /var/www/data/dd588376-1fd5-4158-b3d2-961dc842e9f3/odm_georeferencing/coords.txt
[WARNING] Could not generate coordinates file. Ignore if there is a GCP file. Error: Error in OdmExtractUtm:
No valid GPS position found
For more detailed information, see log file.

[WARNING] Georeferencing failed. Make sure your photos have geotags in the EXIF or you have provided a GCP file. 
[INFO]    Running ODM Georeferencing Cell - Finished
[INFO]    Running ODM Orthophoto Cell
[DEBUG]   running /code/build/bin/odm_orthophoto -inputFile /var/www/data/dd588376-1fd5-4158-b3d2-961dc842e9f3/odm_texturing/odm_textured_model.obj -logFile /var/www/data/dd588376-1fd5-4158-b3d2-961dc842e9f3/odm_orthophoto/odm_orthophoto_log.txt -outputFile /var/www/data/dd588376-1fd5-4158-b3d2-961dc842e9f3/odm_orthophoto/odm_orthophoto.png -resolution 20.0  -outputCornerFile /var/www/data/dd588376-1fd5-4158-b3d2-961dc842e9f3/odm_orthophoto/odm_orthophoto_corners.txt
    plasm.execute(niter=1)
  File "/code/scripts/odm_orthophoto.py", line 67, in process
    '-outputCornerFile {corners}'.format(**kwargs))
  File "/code/opendm/system.py", line 28, in run
    raise Exception("Child returned {}".format(retcode))
Exception: Child returned 1

**Comment by pierotofy** [comment]
Long story short, being that the images are not georeferenced, the model is without a reference scale and the orthophoto tries to generate a really large image (which does not fit in memory).

I was able to process the dataset by lowering the --resolution parameter to something around 4 (down from 20).

![image](https://cloud.githubusercontent.com/assets/1951843/26222595/a686d1e4-3be8-11e7-8590-1b9fc8228379.png)

I've also opened a PR to return a more meaningful error message next time this happens.
